pep-0370.txt

```diff
@@ -5,6 +5,7 @@ Last-Modified: $Date$
 Author: Christian Heimes <christian@python.org>
 Status: Final
 Type: Standards Track
+Topic: Packaging
```

Comment:
I'd exclude this -- it's most about the default behaviour of site and the import system, rather than a Packaging interoperability standard or so.